InfiniteBlanK3T.Calculator/CalculatorLibrary.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+
+namespace CalculatorProgram
+{
+    class Calculator
+    {
+        private List<double> _results;
+        public Calculator()
+        {
+            _results = new List<Double>();
+        }
+        public double DoOperationWithTwoOrMany(List<double> num, string op)
+        {
+            double result = double.NaN; 
+            // Use a switch statement to do the math.
+            switch (op)
+            {
+                case "a":
+                    result = 0;
+                    foreach (double nub in num)
+                    {
+                        result += nub;
+                    }
+                    break;
+                case "s":
+                    result = num[0];
+                    for (int i = 1; i < num.Count; i++)
+                    {
+                        result -= num[i];
+                    }
+                    break;
+                case "m":
+                    result = num[0];
+                    for (int i = 1; i < num.Count; i++)
+                    {
+                        result *= num[i];
+                    }
+                    break;
+                case "p":
+                    if ( num.Count == 2)
+                    {
+                        result = Math.Pow(num[0], num[1]);
+                    }
+                    else
+                    {
+                        result = double.NaN;
+                    }
+                    break;
+                case "d":
+                    // Ask the user to enter a non-zero divisor.
+                    if (num[-1] != 0)
+                    {
+                        result = num[0];
+                        for (int i = 0; i < num.Count; i++)
+                        {
+                            result /= num[i];
+                        }
+                    }
+                    break;
+                // Return text for an incorrect option entry.
+                default:
+                    break;
+            }
+            return result;
```

Comment:
You need to return an error message and redirect me to the menu when I type anything other than valid options. The example below shouldn't be allowed: 
![image](https://user-images.githubusercontent.com/108711476/211977363-34c90fbf-f5cb-42a5-ba00-2ed2d249bc4d.png)